environments/perl/README.md

```diff
@@ -0,0 +1,14 @@
+# Fission Perl environment
+
+This environment is based on Dancer2 and Twiggy.
+
+You can use AnyEvent here.
```

Comment:
I am not familiar with Perl, so AnyEvent, Dancer2 or Twiggy do not say much. Could you maybe make this a bit more descriptive and/or add links?